pkg/util/util.go

```diff
@@ -144,3 +145,16 @@ func IsReadOnlyDisk(disk *v1.Disk) bool {
 
 	return isReadOnlyCDRom
 }
+
+func AlignImageSize(size int64, logger *log.FilteredLogger) int64 {
```

Comment:
Maybe a unit test for this function is valuable

Reply:
Yeah, it's so simple that I wasn't sure it needed its own test...
All it really does is return `x - x % 4096`. But I'm fine adding test(s) if you think that makes sense.

Reply:
Further to the previous comment:
> Maybe replace 67108864 with AlignImageSize(size)?

> Using a static int effectively tests AlignImageSize :)

I think that you de-facto unit test this function but inside a different test :) I think it's better to separate the two for better cohesiveness, but I wouldn't insist on that or anything. Just my point of view :smiley: